modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-retrofit2/infrastructure/ApiClient.kt.mustache

```diff
@@ -60,6 +60,28 @@ import okhttp3.MediaType.Companion.toMediaType
     private val okHttpClientBuilder: OkHttpClient.Builder? = null{{^kotlinx_serialization}},
     private val serializerBuilder: {{#gson}}Gson{{/gson}}{{#moshi}}Moshi.{{/moshi}}Builder = Serializer.{{#gson}}gson{{/gson}}{{#moshi}}moshi{{/moshi}}Builder{{/kotlinx_serialization}},
     private val callFactory : Call.Factory? = null,
+    private val converterFactories: List<Converter.Factory> = listOf(
+        ScalarsConverterFactory.create(),
+        {{#gson}}
+        GsonConverterFactory.create(serializerBuilder.create()),
+        {{/gson}}
+        {{#useRxJava}}
+        RxJavaCallAdapterFactory.create(),
+        {{/useRxJava}}
+        {{#useRxJava2}}
+        RxJava2CallAdapterFactory.create(),
+        {{/useRxJava2}}
+        {{#useRxJava3}}
+        RxJava3CallAdapterFactory.create(),
+        {{/useRxJava3}}
+        {{#moshi}}
+        MoshiConverterFactory.create(serializerBuilder.build()),
+        {{/moshi}}
+        {{#kotlinx_serialization}}
+        kotlinxSerializationJson.asConverterFactory("application/json".toMediaType()),
+        {{/kotlinx_serialization}}
+    ),
+    @Deprecated("this property is deprecated. use converterFactories.")
```

Comment:
Hey @dylan-kwon, thanks for creating this PR.
Could you please change the deprecation to something like this, that shows an error while trying to use converterFactory?
`@Deprecated("This property is deprecated. Please use converterFactories", replaceWith = ReplaceWith("converterFactories"), level = DeprecationLevel.ERROR)`
Thanks